packages/components/organisms/f-cookie-banner/package.json

```diff
@@ -51,18 +51,18 @@
     "extends @justeat/browserslist-config-fozzie"
   ],
   "dependencies": {
-    "@justeat/f-services": "1.14.0"
+    "@justeat/f-services": "1.x"
```

Comment:
is there any reason this shouldn't be a peer & dev dependency?

Reply:
Yeah I was wondering this. It's something I wanted to look into as part of the f-services ticket that's in sprint 👍🏼.